packages/tokens/src/tokens.json

```diff
@@ -2099,4 +2143,4 @@
       "UI Semantic"
     ]
   }
-}
+}
```

Comment:
@Vahid1919 is the missing new line will be an issue?

Reply:
I doubt it will be an issue. I didn't really touch anything here, just copy pasted the file generated by Token Studio. So I would assume it's fine to leave it as is.

Reply:
I had this in some files, where my linter removed it and it broke something afterwards... Happened in the package.json...
But if the quality gate does not throw any errors while linting, its fine.